dev-server/templates/index.mustache

```diff
@@ -27,6 +28,15 @@
     <li><a href="/pdf/widdershins">Widdershins (Wikipedia)</a></li>
   </ul>
 
+  <h2>Test z-index</h2>
```

Comment:
Could you put this on a separate page with a link on the homepage pointing to it? I'd like to avoid cluttering the homepage of the dev server. The test page could be headed "Web page quirks" or something like that.

Reply:
Sure